lib/sass.js

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+import { readdir, writeFile } from 'fs/promises';
+
+import sass from 'sass';
+
+async function compileStyles(src, dst) {
+  const styleFiles = await readdir(src);
+  const promises = styleFiles
+    .filter(filename => filename.endsWith('.scss') && !filename.startsWith('_'))
+    .map(async function(filename) {
+      const styleFileName = filename.slice(0, filename.length - '.scss'.length);
+      // weirdly, renderSync is faster than render with Dart sass: https://sass-lang.com/documentation/js-api#fiber
```

Comment:
I don't think this comment is useful in our code?

Reply:
At some point I'm going to wonder why we decided to use a synchronous method instead of an async one, and this is here to remind me why.

Reply:
ah, noted